pkg/vm/stack/Int.go

```diff
@@ -111,3 +111,27 @@ func (i *Int) Abs() (*Int, error) {
 
 	return b, nil
 }
+
+// Lt returns a bool value from the comparison of two integers, a and b.
+// value is true if a < b.
+// value is false if a >= b.
+func (i *Int) Lt(s *Int) bool {
+	res := i.Value().Cmp(s.Value())
```

Comment:
This can be shortened to `return i.Value().Cmp(s.Value()) == -1`

Reply:
:+1: